src/javascripts/ng-admin/Crud/column/maColumn.js

```diff
@@ -36,7 +36,9 @@ function maColumn($state, $anchorScroll, $compile, Configuration, FieldViewConfi
             scope.datastore = scope.datastore();
             scope.field = scope.field();
             scope.entry = scope.entry();
-            scope.value = scope.entry.values[scope.field.name()];
+            scope.value = typeof scope.entry === 'undefined'
+                ? ''
+                : scope.entry.values[scope.field.name()];
```

Comment:
it's not that .values is undefined, it's that .entry is undefined, so the one-liner will still end up with `Uncaught TypeError: Cannot read property 'values' of undefined`